app/Tquery/OpenApi/OpenApiPost.php

```diff
@@ -62,6 +63,15 @@ public function __construct(
                 )
             ),
             tags: [$tag],
+            parameters: $isFacility ? [
```

Comment:
Nie wiem czy nie lepiej by było to do jakiegoś helpera wyciągnąć i każdorazowo pisać

`parameters: [facilityParameter(), others]`

właśnei ze względu na to others - z isFacility ustawiamy cały array a zaraz się okaże, że np w jednym facility poście trzeba inny param i się zrobi skomplikowana logika tutaj.

Reply:
poprawione tu i tam